src/ngResource/resource.js

```diff
@@ -701,12 +701,9 @@ angular.module('ngResource', ['ng']).
               response.resource = value;
 
               return response;
-            }, function(response) {
-              (error || noop)(response);
-              return $q.reject(response);
             });
 
-            promise['finally'](function() {
+            promise = promise['finally'](function() {
```

Comment:
OOC, does this make any difference (in this particular case, where the code inside the `finally` callback is synchronous) ?